lib/migration.js

```diff
@@ -66,28 +66,6 @@ function mixinMigration(MsSQL) {
     });
   };
 
-  MsSQL.prototype.isActual = function(cb) {
-    var ok = false;
-    var self = this;
-    async.each(Object.keys(this._models), function(model, done) {
-      self.getTableStatus(model, function(err, fields, indexes) {
-        self.alterTable(model, fields, indexes, function(err, needAlter) {
```

Comment:
Fixed in commit https://github.com/strongloop/loopback-connector-mssql/pull/129/commits/002c2f081b7d4c0bab3a0b2bb85d4e9601648dec